app/helpers/github_webhook_helper.rb

```diff
@@ -0,0 +1,39 @@
+module GithubWebhookHelper
```

Comment:
As written this helper is very tightly coupled to the controller. Generally we prefer to keep any business logic out of the controller, as it's very difficult to unit-test a controller (there's a lot of setup of complicated objects like the request, params, etc) and we've frequently found that business logic needs to be re-used in other contexts (e.g. other controllers, or even rake tasks or background jobs).

Can you refactor these helper functions out into a class / classes that are called from the controller, but could equally easily be called from a rake task or other entry point?

Reply:
@undergroundwebdesigns Thanks for all the amazing feedback! I have made all the changes and refactored the `github_webhook_helper` to a more decoupled `github_service`. Let me know your thoughts. The reason I decided to couple it with the controller was to use the existing controller method JSON responses. However, I agree with your point about tests as these may change over time and the helper methods may break as writing tests would be difficult for the `github_webhook` route.